Kconfig

```diff
@@ -1,5 +1,6 @@
 # Kconfig file for LVGL v8.0
```

Comment:
nit: not sure mentioning the version here (still) makes sense?

Reply:
not part of the change :)

Reply:
yeah guess that's upstream anyway, not worth diverging to fix it

Reply:
Yeah its a leftover of upstream, will add it to the PR in there. Will be in our fork with v9 :^)